Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -332,6 +332,7 @@ - (void)addMessageCall:(OCTCall *)call
     OCTMessageCall *messageCall = [OCTMessageCall new];
     messageCall.callDuration = call.callDuration;
     messageCall.callEvent = event;
+    messageCall.isOutgoing = call.isOutgoing;
```

Comment:
Instead of adding `isOutgoing` to OCTMessageCall you should set `sender` property of OCTMessageAbstract (to friend or to nil, if message is outgoing).

I suggest you
- remove `isOutgoing` from OCTCall
- add `caller` property instead (which will be nil in case of outgoing call)
- add `isOutgoing` method as it is done is OCTMessageAbstract
- when creating message from call simply assign `caller` to `sender`

This way we would have similar interface in messages and calls. Also for the future we would already have `caller` when we'll be implementing group calls.